src/isar/services/readers/map_reader.py

```diff
@@ -38,12 +36,7 @@ def get_predefined_map_configs(self) -> Optional[dict]:
         for file in json_files:
             path_to_file = self.predefined_map_config_folder.joinpath(file.name)
             map_config: MapConfig = self.get_map_config(path_to_file)
-            if map_config is None:
-                logger.warning(
-                    f"File {path_to_file.as_posix()}"
-                    f" could not be parsed to a map_config"
-                )
-            elif map_config.map_name in map_configs:
+            if map_config.map_name in map_configs:
                 logger.warning(
                     f"Map name {map_config.map_name} : {path_to_file.as_posix()}"
                     f" already exist. Skipping both original and duplicate ..."
```

Comment:
In the bottom of this file there is a function:
`def get_map_config_by_name(...`
Think we can remove the try/except and the Optional part of the return typing